mcs/class/System/mobile_System.dll.sources

```diff
@@ -305,7 +305,6 @@ Mono.Btls/MonoBtlsX509LookupMonoCollection.cs
 Mono.Btls/MonoBtlsX509LookupMono.cs
 Mono.Btls/MonoBtlsX509LookupType.cs
 Mono.Btls/MonoBtlsX509Name.cs
-Mono.Btls/MonoBtlsX509NameList.cs
```

Comment:
don't remove this from the System sources, it'd break monodroid. The files have #ifdefs in them so they're only compiled when BTLS is enabled.